skyportal/handlers/api/spectrum.py

```diff
@@ -47,7 +47,9 @@
 class SpectrumHandler(BaseHandler):
     def get_groups(self, group_ids):
         groups = []
-        if group_ids == "all":
```

Comment:
Shouldn't this return an error? Or what is the context where we need this to return None?

Reply:
one of the usages tests that groups is True, but I think that it may be easier to check if group_id is not None or empty.